pkg/storage/SQL/operational_solution/and_possible_get_by_operational_need_id_LOADER.sql

```diff
@@ -13,9 +13,9 @@ possibleSolution AS (
     /*Get the mapped possible solutions for given operational need */
     SELECT
         OpSol.id,
-        qID.operational_need_id AS operational_need_id,
+        qID.operational_need_id,
         pOpSol.id AS solution_type,
-        OpSol.needed AS needed,
```

Comment:
Were these just "useless" instances of name casting? (at least as far as `sqlx` is concerned?)

Reply:
exactly! I have a sql fluff linter installed locally for VSCode. It suggested you shouldn't self alias, so I just implemented it.